src/camera_system.py

```diff
@@ -14,4 +15,6 @@ def __init__(self) -> None:
 
     def start(self) -> None:
         """ゲーム攻略を計画する."""
-        pass
+        # キャリブレーション後に走行体状況監視モジュールを実行する
+        client = Client()
```

Comment:
今後どうなるかわからんけど、できれば実行時引数で渡したいからできるだけ上のクラスで設定したい
```suggestion
        server_ip = "192.168.11.17:8000"
        client = Client(server_ip)
```